solana/program_v2/packages/tests/package.json

```diff
@@ -3,18 +3,19 @@
   "version": "1.0.0",
   "license": "MIT",
   "scripts": {
-    "build": "tsc --build",
-    "lint": "eslint src/**/*.ts"
+    "build": "tsc --build"
   },
   "dependencies": {
-    "@identity.com/gateway-solana-client": "^2.0.0-beta.8",
-    "@identity.com/gateway-solana-idl": "^2.0.0-beta.8",
-    "@project-serum/anchor": "^0.26.0",
-    "@solana/spl-token": "^0.3.5",
-    "@solana/web3.js": "^1.41.3"
+    "@identity.com/gateway-solana-client": "^2.0.0-beta.12",
```

Comment:
I think technically everything in the `test` project should be a `devDependency` :)